person.rb

```diff
@@ -0,0 +1,24 @@
+# implementation person class
+class Person
+  def initialize(age, _parent_permission: true, name: 'Unknown')
+    @id = id
```

Comment:
- [OPTIONAL] To make @id value unique, you can use random generated numbers like so: 
`@id = Random.rand(1..1000)`, please kindly fix that.